docs/backend-api/resources/fleet/vehicle/service_task/service_task.md

```diff
@@ -342,13 +426,24 @@ Report file.
 
 ### `list`
 
-List all service works of all user vehicles.
+Lists the service works, sorted in ascending order by the predicted date
```

Comment:
invalid, sorting is determined by `sort` field. `predicted_date` is one of `sort` field values

Reply:
list all the values that can be specified in the sort field. probably better in the field description